scripts/sct_get_centerline.py

```diff
@@ -4,135 +4,157 @@
-                      type_value='int',
-                      description='Degree of smoothing for centerline fitting. Only for -centerline-algo {bspline, linear}.',
-                      mandatory=False,
-                      default_value=30)
-    parser.add_option(name="-o",
-                      type_value='file_output',
-                      description='File name (without extension) for the centerline output files. By default, output'
-                                  'file will be the input with suffix "_centerline"',
-                      mandatory=False,
-                      example="centerline_optic")
-    parser.add_option(name="-gap",
-                      type_value="float",
-                      description="Gap in mm between manually selected points. Only with method=viewer.",
-                      mandatory=False,
-                      default_value='20.0')
-    parser.add_option(name="-igt",
-                      type_value="image_nifti",
-                      description="File name of ground-truth centerline or segmentation (binary nifti).",
-                      mandatory=False)
-    parser.add_option(name="-v",
-                      type_value="multiple_choice",
-                      description="1: display on, 0: display off (default)",
-                      mandatory=False,
-                      example=["0", "1", "2"],
-                      default_value="1")
-    parser.add_option(name='-qc',
-                      type_value='folder_creation',
-                      description='The path where the quality control generated content will be saved',
-                      default_value=None)
-    parser.add_option(name='-qc-dataset',
-                      type_value='str',
-                      description='If provided, this string will be mentioned in the QC report as the dataset the process was run on',
-                      )
-    parser.add_option(name='-qc-subject',
-                      type_value='str',
-                      description='If provided, this string will be mentioned in the QC report as the subject the process was run on',
-                      )
+    parser = argparse.ArgumentParser(
+        description=(
+            "This function extracts the spinal cord centerline. Three methods are available: OptiC (automatic), "
+            "Viewer (manual) and Fitseg (applied on segmented image). These functions output (i) a NIFTI file with "
+            "labels corresponding to the discrete centerline, and (ii) a csv file containing the float (more precise) "
+            "coordinates of the centerline in the RPI orientation. \n"
+            "\n"
+            "Reference: C Gros, B De Leener, et al. Automatic spinal cord localization, robust to MRI contrast using "
+            "global curve optimization (2017). doi.org/10.1016/j.media.2017.12.001"
+        ),
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
```

Comment:
is the verdict in on whether this is needed?

Reply:
I got a :+1: in https://github.com/neuropoly/spinalcordtoolbox/pull/2819#discussion_r476097347 but feel free to let me know what you think about the reasoning, too. :slightly_smiling_face:

Reply:
i haven't tested-- if you test it and see that the ".py" is present without this line of code, then it should be kept